fenix/app/src/main/res/values/strings.xml

```diff
@@ -2358,6 +2364,8 @@
     <string name="translation_error_language_not_supported_warning_text">Sorry, we don’t support %1$s yet.</string>
     <!-- Button text on the warning card when a language is not supported. The link will take the user to a page to a support page about translations. -->
     <string name="translation_error_language_not_supported_learn_more">Learn more</string>
+    <!-- Snackbar title shown if the user closes the Translation Request dialogue and a translation is in progress. -->
+    <string name="translation_in_progress_snackbar">Translating...</string>
```

Comment:
```suggestion
    <string name="translation_in_progress_snackbar">Translating…</string>
```
I think we should be using the "…" text encoding here just like in https://searchfox.org/mozilla-mobile/source/firefox-android/fenix/app/src/main/res/values/strings.xml#1254. Very rarely is this ever gonna be 3 "." for localization purposes.

Reply:
I also saw we used the unicode in https://searchfox.org/mozilla-mobile/source/firefox-android/fenix/app/src/main/res/values/strings.xml#1409. @Delphine what would you advise here?

Reply:
Done